spec/models/fika_goer_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe FikaGoer, type: :model do
+  describe 'DB Table' do
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :start_date }
+    it { is_expected.to have_db_column :management }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_presence_of :email }
+    it { is_expected.to validate_presence_of :start_date }
+    it { should allow_value(true).for(:management) }
+    it { should allow_value(false).for(:management) }
```

Comment:
gold star

Reply:
If of interest, the recommendation from the [Shoulda-Matchers docs](https://matchers.shoulda.io/docs/v5.0.0/Shoulda/Matchers/ActiveModel.html#validate_inclusion_of-instance_method) advises against testing for the inclusion of booleans:
```
We discourage using validate_inclusion_of with boolean columns. In fact, there is never a case where a boolean column will be anything but true, false, or nil, as ActiveRecord will type-cast an incoming value to one of these three values. That means there isn't any way we can refute this logic in a test.
```